common/include/pcl/TextureMesh.h

```diff
@@ -91,9 +91,17 @@ namespace pcl
     pcl::PCLHeader  header;
 
 
-    std::vector<std::vector<pcl::Vertices> >    tex_polygons;     // polygon which is mapped with specific texture defined in TexMaterial
-    std::vector<std::vector<Eigen::Vector2f, Eigen::aligned_allocator<Eigen::Vector2f> > > tex_coordinates;  // UV coordinates
-    std::vector<pcl::TexMaterial>               tex_materials;    // define texture material
+    /** \brief polygon which is mapped with specific texture defined in TexMaterial */
+    std::vector<std::vector<pcl::Vertices> >    tex_polygons;
+    /** \brief UV coordinates */
+    std::vector<std::vector<Eigen::Vector2f, Eigen::aligned_allocator<Eigen::Vector2f> > > tex_coordinates;
```

Comment:
Does Eigen have a typedef for a Vector of Eigen-types?

Reply:
No, as far as I know